agent/tool-scripts/pprof

```diff
@@ -147,6 +147,7 @@ ose_pprof() {
 collect_data() {
     while true; do
             $tool_package tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_.txt http://localhost:6060/debug/pprof/profile
+            $tool_package tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_mem.txt http://localhost:6060/debug/pprof/heap
```

Comment:
So won't this change the collection internal so that on the first interval the normal profile is collected, and then the second interval the heap is collected?

Or do the tools run in parallel some how?